chrome/content/newtab.css

```diff
@@ -100,7 +100,7 @@ url-prefix("about:home") {
     border-radius: 12px !important;
     height: 44px !important;
     width: 44px !important;
-    box-shadow: none !important;
+    box-shadow: rgba(0, 0, 0, 0.1) 0px 2px 4px !important;
```

Comment:
Use the newer CSS colour syntax: `rgb(0 0 0 / .1)` to match the rest of the theme.